src/documents/CoreAPI.md

```diff
@@ -0,0 +1,635 @@
+---
+title: Core API
+layout: default
+category: Infusion
+---
+
+# Infusion Core API
+
+This file documents some low-level APIs which are provided by Infusion at the JavaScript level. Note that since
```

Comment:
The ToC widget is freely available for users of the "display preferences" widget installed at the top of the page! And I fear that if a literal ToC was added to the markup, it would end up confusingly duplicating the output from the widget.